e2etest/get_related_places_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2etest
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"path"
+	"runtime"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/proto"
+	"github.com/datacommonsorg/mixer/server"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestGetRelatedPlacesTest(t *testing.T) {
+	ctx := context.Background()
+	client, err := setup(server.NewMemcache(map[string][]byte{}))
+	if err != nil {
+		t.Fatalf("Failed to set up mixer and client")
+	}
+	_, filename, _, _ := runtime.Caller(0)
+	goldenPath := path.Join(
+		path.Dir(filename), "../golden_response/staging/get_related_places")
+
+	for _, c := range []struct {
+		goldenFile string
+		dcids      []string
+		popType    string
+		mprop      string
+		statType   string
+		pvs        map[string]string
+	}{
+		{
+			"population.json",
+			[]string{"geoId/06085"},
+			"Person",
+			"count",
+			"measuredValue",
+			nil,
+		},
+		{
+			"income.json",
+			[]string{"geoId/06085"},
+			"Person",
+			"income",
+			"medianValue",
+			map[string]string{
+				"age":          "Years15Onwards",
+				"incomeStatus": "WithIncome",
+			},
+		},
+		{
+			"age.json",
+			[]string{"geoId/06085"},
+			"Person",
+			"age",
+			"medianValue",
+			nil,
+		},
+		{
+			"unemployment.json",
+			[]string{"geoId/06085"},
+			"Person",
+			"unemploymentRate",
+			"measuredValue",
+			nil,
+		},
+		{
+			"crime.json",
+			[]string{"geoId/06"},
+			"CriminalActivities",
+			"count",
+			"measuredValue",
+			map[string]string{
+				"crimeType": "UCR_CombinedCrime",
+			},
+		},
+	} {
+		// TODO: remove this when dataset is updated.
+		continue
```

Comment:
Remove this?

Reply:
if remove test won't pass. Need to use the new dataset.